<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KnifeForm</class>
 <widget class="QWidget" name="KnifeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>307</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="openList">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Images.qrc">
         <normaloff>:/Images/More.png</normaloff>:/Images/More.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="editBTN">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Images.qrc">
         <normaloff>:/Images/Edit.png</normaloff>:/Images/Edit.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="enlBTN">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>24</horstretch>
         <verstretch>24</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Images.qrc">
         <normaloff>:/Images/Enlarge.png</normaloff>:/Images/Enlarge.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delBTN">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Images.qrc">
         <normaloff>:/Images/Delete.png</normaloff>:/Images/Delete.png</iconset>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>266</width>
        <height>176</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="Numbers">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="Name">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Название</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Image_place">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
